Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -64,7 +64,7 @@ void GeneralWidget::CreateWidgets()
 
   m_video_box->setLayout(m_video_layout);
 
-  for (auto& backend : g_available_video_backends)
+  for (auto& backend : VideoBackendBase::GetAvailableBackends())
     m_backend_combo->addItem(tr(backend->GetDisplayName().c_str()),
                              QVariant(QString::fromStdString(backend->GetName())));
```

Comment:
Do we want to add some curlies here while we're at it, since the contents of the loop span over multiple lines?